tree/treeplayer/src/TTreeProcessorMT.cxx

```diff
@@ -525,43 +525,35 @@ void TTreeProcessorMT::Process(std::function<void(TTreeReader &)> func)
                                                           fFileNames, clusterAndEntries.second);
    }
 
-   const auto &clusters = clusterAndEntries.first;
-   const auto &entries = clusterAndEntries.second;
-
-   // Retrieve number of entries for each file for each friend tree
-   const auto friendEntries = hasFriends ? GetFriendEntries(fFriendInfo) : std::vector<std::vector<Long64_t>>{};
-
    // Parent task, spawns tasks that process each of the entry clusters for each input file
```

Comment:
this comment is a bit misplaced now, maybe
```suggestion
   // Per-file processing in case we retrieved all cluster info upfront
```

and similarly the other lambda should have a comment that says (something like) "Per-file processing that also retrieves cluster info for a file"